.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+# General Information
+
+1. **What change/s does this pull request introduce? (Provide a detailed explanation)**
+
+2. **Does this pull request break something? What?**
+
+# Testing
+
+1. **Was this change tested locally?**
+
+2. **Was this change tested in staging?**
+
+3. **Was this change tested in production?**
+
+# Dependencies
+
+1. **Does this have any dependency? (link PRs)**
```

Comment:
- Could we add an item on release-notes? e.g. Ask for a one-line summary of any user-facing changes.
- It may be a good idea to add an item for code-style (e.g. [go-report-card ](https://goreportcard.com/) for GO code).

Reply:
Additional items to consider:
- Commits: are all the commits necessary (e.g. all describe a coherent set of changes)? or can they be squashed? 
- Is this a single change? Or should it be broken into multiple PRs?

Reply:
For the first part, I think that's more for the release notes. Do you think we should separte the questions of features added and broken stuff from internal and user sides?